bin/emulator/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(TARGET_NAME emulator_wrapper)
+set(SOURCES
+    wrapper.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/../assigner/src/utils.cpp)
+
+set(CMAKE_CXX_STANDARD 23)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_library(${TARGET_NAME} SHARED ${SOURCES})
```

Comment:
If this code produces a library, why is it in `bin/` directory then? I guess, it should go to `libs/` right?

Reply:
This library is not exactly library;), just an wrapper special for call from GO

Reply:
But this target produces an `.so` file anyway, right?

Reply:
yes, but we want use it only in `node.go`. Do you think it make sense move `wrapper` to `libs`